exercises/stringify-to-demystify/problem/problem.md

```diff
@@ -0,0 +1,47 @@
+`__toString()` a class magic method long-standing in PHP but never truly something you could rely on unless you rolled your own interface. All that has changed with the simple introduction of the `Stringable` interface in PHP 8.
+
+A simple interface that requires the implementation of `__toString(): string`
```

Comment:
`Stringable` is a

It seems a bit detached from the above sentence and needs repeating I think